Common/Inc/Interchip.h

```diff
@@ -1,17 +1,12 @@
 /*
 * Interchip packets
-* Packets should be a multiple of 16 bits
 */
 #ifndef INTERCHIP_H
 #define INTERCHIP_H
-typedef struct {
-	int16_t PWM[12];
-	uint16_t autonomous_level;
-} Interchip_StoA_Packet;    //Safety to Autopilot packet
 
 typedef struct {
+	uint16_t safetyLevel;
 	int16_t PWM[12];
-	uint16_t autonomous_level;
-} Interchip_AtoS_Packet;    //Autopilot to Safety packet
+} Interchip_Packet;  
 
 #endif
```

Comment:
newline . See if there are any other places where the "do not enter" sign appears (we really need to automate this lol)